api/views/submission.py

```diff
@@ -270,20 +270,20 @@ def _should_check_transcription(volunteer: BlossomUser) -> bool:
         """
         probabilities = [
             (5, 1),
-            (50, 0.8),
-            (100, 0.7),
-            (250, 0.6),
-            (500, 0.5),
-            (1000, 0.3),
-            (5000, 0.1),
+            (50, 0.4),
+            (100, 0.2),
+            (250, 0.1),
+            (500, 0.05),
+            (1000, 0.02),
+            (5000, 0.01),
```

Comment:
Looks like there's some disparity between these values and the docstring above.

I'd argue it would be better to leave a comment noting the structure of this and then leave the actual values of `level => probablity` mapping exclusively in the code to reduce maintenance overhead.

Reply:
crap, I did completely forget about the comment

Reply:
Fixed!